src/read.c

```diff
@@ -2756,6 +2762,23 @@ static avifResult avifParseItemPropertyAssociation(avifMeta * meta, const uint8_
             // Copy property to item
             const avifProperty * srcProp = &meta->properties.prop[propertyIndex];
 
+            // ISO/IEC 23000-22 Section 6.5.1:
+            //   All transformative properties associated with coded and derived images shall be marked as essential,
+            //   and shall be from the set defined in 7.3.6.7 or the applicable MIAF profile. No other essential
+            //   transformative property shall be associated with such images.
+            const avifBool isTransformative = !memcmp(srcProp->type, "clap", 4) || !memcmp(srcProp->type, "irot", 4) ||
+                                              !memcmp(srcProp->type, "imir", 4);
+            // ISO/IEC 23008-12 Section 3.1.29:
```

Comment:
3.1.29 => 3.1.28

Alternatively, include the version/edition number of ISO/IEC 23008-12.

In ISO/IEC 23008-12:2022 (the [current published version](https://www.iso.org/standard/83650.html)), the term "item property" is defined in 3.1.28.